client/makelunch.js

```diff
@@ -32,7 +42,7 @@ Meteor.startup(function () {
       path:'/editmeal/:_id',
       data: function () {
         var meal = Meals.findOne(this.params._id)
-        console.log(meal)
+        if (!meal) return false
```

Comment:
I'm not sure how the template will handle bad ids which do not correspond to actual meals here, but it would seem better to return an object with null entries for `meal` and `eaters` should we ever want to interact with the data context without having to check that these keys exist.